gp-translation-extended-api/gp-translation-extended-api.php

```diff
@@ -290,6 +341,8 @@ function init_new_routes() {
 		GP::$router->add( '/translations/(\d+)/-set-status', array( 'GP_Route_Translation_Extended', 'translations_options_ok' ), 'options' );
 		GP::$router->add( '/translations/-query-by-originals', array( 'GP_Route_Translation_Extended', 'translations_get_by_originals' ), 'post' );
 		GP::$router->add( '/translations/-query-by-originals', array( 'GP_Route_Translation_Extended', 'translations_options_ok' ), 'options' );
+		GP::$router->add( '/translations/-translation-set', array( 'GP_Route_Translation_Extended', 'translation_get_untranslated_language' ), 'get' );
```

Comment:
Maybe rename `translation-set` to something that makes more sense when not speaking the GP lingo - maybe just something like `by-locale` or similar?